Makefile.Common

```diff
@@ -4,7 +4,7 @@ SRC_ROOT := $(realpath $(dir $(lastword $(MAKEFILE_LIST))))
 # build tags required by any component should be defined as an independent variables and later added to GO_BUILD_TAGS below
 GO_BUILD_TAGS=""
 GOTEST_OPT?= -race -v -timeout 300s --tags=$(GO_BUILD_TAGS)
-GOTEST_INTEGRATION_OPT?= -race -timeout 120s
+GOTEST_INTEGRATION_OPT?= -race -timeout 300s
```

Comment:
Can we try 3 minutes first?

Reply:
I was expecting the test to fail at 5 minutes, but if it passes, I can drop it down to 3 minutes for another test run.

Reply:
The successful test run took 289s, and the test failed with a 3 minute timeout. I think 5 minutes is a good number for now, but if we see failures with any frequency we can increase the timeout as well.

Reply:
Ok. Sounds good